scripts/performance_monitor.py

```
@@ -0,0 +1,252 @@
+import subprocess
+import time
+import json
+import signal
+import sys
+from datetime import datetime
+from pathlib import Path
+import threading
+import os
+
+class PerformanceMonitor:
+    def __init__(self, log_dir="/tmp/claude-performance-logs"):
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/claude-performance-logs"

(S108)